src/supa/db/model.py

```diff
@@ -437,28 +437,30 @@ class Parameter(Base):
     value = Column(Text)
 
 
-class Port(Base):
-    """DB mapping for ports (STPs) from the Orchestrator."""
+class Topology(Base):
+    """DB mapping for STP's and ports in the topology from the NRM."""
 
-    __tablename__ = "ports"
+    __tablename__ = "topology"
 
-    port_id = Column(Uuid, primary_key=True, comment="subscription_id of a port in the Orchestrator")
-    name = Column(Text, nullable=False, unique=True, index=True)
+    stp_id = Column(Text, primary_key=True, index=True)
```

Comment:
I wonder if this will map to Postgres, it feels a bit weird to have a Text id...

Also: worth investigating if it doesn't add an id by default: I think SQLAlchemy has a setting for it.

Reply:
No automatic ID field: but implicit auto increment when you use integer.

https://stackoverflow.com/questions/20848300/unable-to-create-autoincrementing-primary-key-with-flask-sqlalchemy

Probably good to switch to postgres when possible -> you can then use UUID's without the need to specify them in the INSERT.